AllCoreDevs-Meetings/Meeting 146.md

```diff
@@ -0,0 +1,174 @@
+
+**Micah Zoltu**
+* I feel like we've been putting off planning Shanghai forever. Like if we've got everybody here for an hour, is there a reason we're not starting to plan Shanghai? 
+
+**Ansgar Dietrichs**
+* Well let me just ask what's what's the, do people have opinions and whether we should use the time today to talk about Shanghai or I'll leave it for a later day? 
+
+**Trent**
+* It In theory is, and I said, yeah, but I don't think people are prepared to do that today. 
+
+**Lukasz**
+* Yes. Very little participation. People are also competing. 
+
+**Ansgar Dietrichs**
+* Okay. Yeah. Then, I think, I mean, I can, I can just very briefly say because I've been somewhat involved with the kind of EIP 4844 for effort. So yeah, again, I don't think it's the best time to actually discuss the details of this. People are still working on this. so this is, you know, this, first kind of version of this, people are still trying to get it ready so that it can be considered an included, for Shanghai. So this is a very active effort. And if any, any of the, of the client teams, happened to have some return now that visibly everything is in place. And if they like bot waiting for the merge, you know, it's always there and we're as happy for, for anyone to reach out and terms of gives them early feedback before we kind of open it wide up after the merge when everyone is ready. And, but, but yeah, I don't, I don't think today's the day to really dive into this. And, I think turned out to just, linked, some, some, some treats for the team about 10 of us, which is another one of these really exciting, updates that are scheduled for, for Shanghai. So, but anyone interested to have a look I'm not scheduled for Shanghai because we still haven't had a meeting to schedule. This is why you need someone with experience. No, nobody. Yeah, exactly. So this is just a considered for tonight, but this is what I should have said. it's, yeah. Considered for inclusion all right. Now. and that is a very important distinction, so yes. Thank you, Micah. but yeah, with that, on last chance, but it sounds like we're a bit, we are, edit close, so thanks everyone for coming. have a good day, have a good time before the merge. 
```

Comment:
```suggestion
* Okay. Yeah. Then, I think, I mean, I can, I can just very briefly say because I've been somewhat involved with the kind of EIP-4844 for effort. So yeah, again, I don't think it's the best time to actually discuss the details of this. People are still working on this. so this is, you know, this, first kind of version of this, people are still trying to get it ready so that it can be considered an included, for Shanghai. So this is a very active effort. And if any, any of the, of the client teams, happened to have some return now that visibly everything is in place. And if they like bot waiting for the merge, you know, it's always there and we're as happy for, for anyone to reach out and terms of gives them early feedback before we kind of open it wide up after the merge when everyone is ready. And, but, but yeah, I don't think today's the day to really dive into this. And, I think turned out to just, linked, some, some, some treats for the team about 10 of us, which is another one of these really exciting, updates that are scheduled for, for Shanghai. So, but anyone interested to have a look I'm not scheduled for Shanghai because we still haven't had a meeting to schedule. This is why you need someone with experience. No, nobody. Yeah, exactly. So this is just a considered for tonight, but this is what I should have said. it's, yeah. Considered for inclusion all right. Now. and that is a very important distinction, so yes. Thank you, Micah. but yeah, with that, on last chance, but it sounds like we're a bit, close, so thanks everyone for coming. have a good day, have a good time before the merge. 
```